pkg/kdump/config-client.js

```diff
@@ -113,21 +124,185 @@ export class ConfigFile {
 
         // make sure we copy the original keys so we overwrite the correct lines when saving
         this._originalSettings = { };
-        Object.keys(this.settings).forEach((key) => {
-            this._originalSettings[key] = { ...this.settings[key] };
+        Object.keys(this.settings._internal).forEach((key) => {
+            this._originalSettings[key] = { ...this.settings._internal[key] };
         });
+
+        this._extractSettings();
+
         if (!skipNotify)
             this.dispatchEvent("kdumpConfigChanged", this.settings);
     }
 
+    /* extract settings managed by cockpit from _internal into platform independent model
+     */
+    _extractSettings() {
+        // "path" applies to all targets
+        const path = this.settings._internal.path || { value: "" };
+
+        Object.keys(this.settings._internal).forEach((key) => {
+            if (key === "nfs") {
+                // split nfs line into server and export parts
+                const parts = this.settings._internal.nfs.value.match(/^([^[][^:]+|\[[^\]]+\]):(.*)$/);
+                if (!parts)
+                    return;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17544-20220719-143653-7a0d5780-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)